source/reference/operator/query.txt

```diff
@@ -338,25 +352,17 @@ Projection Operators
 
      - Description
 
-   * - :projection:`$`
-
-     - Projects the first element in an array that matches the query condition.
-
-   * - :projection:`$elemMatch`
-
-     - Projects the first element in an array that matches the specified :projection:`$elemMatch` condition.
-
-   * - :projection:`$meta`
-
-     - Projects the document's score assigned during :query:`$text` operation.
+   * - :query:`$comment`
 
-   * - :projection:`$slice`
+     - Allows use of commented code within the query language.
```

Comment:
Why change the $comment definition on your $rand ticket? This is not what $comment does ...

Reply:
I didn't make that change on purpos. I must have gotten picked up during an update somehow.

Reply:
Current version looks like this on those lines. There's no mention of $comment: 

   * - :projection:`$meta`

     - Projects the document's score assigned during :query:`$text` operation.

   * - :projection:`$slice`

     - Limits the number of elements projected from an array. Supports skip and limit slices.

I'll put back the version with $slice and $without $comment.

Reply:
Actually it's a lot more messed up than that. There's a bunch of stuff to fix.